### PR TITLE
Changes to add operation id for outbound rules

### PR DESCRIPTION
**Pull request by khannarhea**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by MikhailTryakhov — APPROVED** [review]

**Comment by MikhailTryakhov** [comment]
@kkhannarhea please fix failed tests

**Comment by EvgenyAgafonchikov** [comment]
@khannarhea, could you please change line endings in loadBalancer.json back to LF (now CRLF). Currently it is difficult to find in diff where the change is.

**Comment by EvgenyAgafonchikov** [comment]
@khannarhea, I created PR with some fixes by @MikhailTryakhov's request: https://github.com/khannarhea/azure-rest-api-specs/pull/1. Please merge it to fix CI issues and revert line endings to default

**Comment by khannarhea** [comment]
thanks @EvgenyAgafonchikov  and @MikhailTryakhov  - i have merged the branch

File: specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/loadBalancer.json [hunk 1]
```diff
@@ -800,6 +801,100 @@
...
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```
Comment (RyanBensonMSFT) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/loadBalancer.json]: You should include a 'default' response for all error cases, example: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/msi/resource-manager/Microsoft.ManagedIdentity/preview/2015-08-31-preview/ManagedIdentity.json 

          "default": {
            "description": "Error response describing why the operation failed.",
            "schema": {
              "$ref": "#/definitions/CloudError"
            }
          }

This should be addressed across your entire Swagger spec though. Please include this in your next PR.

**Review by sergey-shandar — APPROVED** [review]

**Review by ryansbenson — APPROVED** [review]
Approved

**Comment by sergey-shandar** [comment]
@khannarhea are you going to address `default` response issues?

File: specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/loadBalancer.json (hunk 1 above)
Comment (khannarhea) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/loadBalancer.json]: Yes, I will address it in the next PR -  Tracking item - https://trello.com/c/HrzSc9bZ/69-include-a-default-response-for-all-error-cases

**Pull request merged**